dipy/reconst/csdeconv.py

```diff
@@ -654,7 +659,11 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7):
     the highest and second highest eigenvalues in the ROI with FA higher than
     threshold. We also include the average S0s.
 
-    Finally, we also return the `ratio` which is used for the SDT models.
+    We also return the `ratio` which is used for the SDT models. If requested,
+    the number of voxels used for estimating the response function is also
+    returned, which can be used to judge the fidelity of the response function.
+    As a rule of thumb, at least 300 voxels should be used to estimate a good
```

Comment:
It is a rule of thumb but it came from Tournier.

Maxime Descoteaux, PhD
Professor
Sherbrooke Connectivity Imaging Laboratory (SCIL)
Centre de Recherche CHUS
Computer Science department
Sherbrooke University
2500 Boul. Université
Sherbrooke, Québec
J1K 2R1, Canada
phone: +1 819 821-8000 x66129
fax: +1 819 821-8200
http://scil.dinf.usherbrooke.ca

Scientific director
Visualization and Image Analysis Plateform (PAVI)
http://pavi.dinf.usherbrooke.ca/

On Tue, Jan 20, 2015 at 12:04 AM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/reconst/csdeconv.py
> <https://github.com/nipy/dipy/pull/561#discussion-diff-23203105>:
>
> > @@ -654,7 +659,11 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7):
> >      the highest and second highest eigenvalues in the ROI with FA higher than
> >      threshold. We also include the average S0s.
> >
> > -    Finally, we also return the `ratio` which is used for the SDT models.
> > +    We also return the `ratio` which is used for the SDT models. If requested,
> > +    the number of voxels used for estimating the response function is also
> > +    returned, which can be used to judge the fidelity of the response function.
> > +    As a rule of thumb, at least 300 voxels should be used to estimate a good
>
> Funny, I always though it was really just a rule of thumb, I'll add the
> proper ref tomorrow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/561/files#r23203105>.[image: Web Bug
> from
> https://github.com/notifications/beacon/ACEgB7D7HD7LNnsNPGNypKYE4dNs7gFGks5njdlWgaJpZM4DTu3P.gif]
>